<ui version="4.0" >
 <class>Mainwindow</class>
 <widget class="QMainWindow" name="Mainwindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>574</height>
   </rect>
  </property>
  <property name="acceptDrops" >
   <bool>false</bool>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled" >
   <bool>true</bool>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="tabShape" >
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="usesScrollButtons" >
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProjekt" >
    <property name="title" >
     <string>Project</string>
    </property>
    <addaction name="actionProjectNew" />
    <addaction name="actionProjectOpen" />
    <addaction name="actionProjectSave_As" />
    <addaction name="actionProjectSave" />
    <addaction name="actionProjectClose" />
    <addaction name="separator" />
    <addaction name="actionProjectAddFile" />
    <addaction name="actionProjectAddCurrentFile" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuDatei" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionFileNew" />
    <addaction name="actionFileOpen" />
    <addaction name="actionFileSaveAs" />
    <addaction name="actionFileSave" />
    <addaction name="actionFileClose" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuBearbeiten" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionEditUndo" />
    <addaction name="actionEditRedo" />
    <addaction name="separator" />
    <addaction name="actionEditCut" />
    <addaction name="actionEditCopy" />
    <addaction name="actionEditPaste" />
    <addaction name="actionActionEditDelete" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuFenster" >
    <property name="title" >
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom" >
     <property name="title" >
      <string>Zoom</string>
     </property>
     <addaction name="actionViewZoom_In" />
     <addaction name="actionZoom_Out" />
     <addaction name="actionViewDefault_Zoom" />
    </widget>
    <widget class="QMenu" name="menuGeben_Sie_Text_ein" >
     <property name="title" >
      <string>Geben Sie Text ein</string>
     </property>
    </widget>
    <addaction name="menuZoom" />
    <addaction name="separator" />
    <addaction name="menuGeben_Sie_Text_ein" />
   </widget>
   <widget class="QMenu" name="menuNavigatoin" >
    <property name="title" >
     <string>Navigation</string>
    </property>
    <addaction name="actionEditFind" />
    <addaction name="actionFind_in_Files" />
   </widget>
   <widget class="QMenu" name="menuBuild" >
    <property name="title" >
     <string>Build</string>
    </property>
    <widget class="QMenu" name="menuSelect_Target" >
     <property name="title" >
      <string>Select Target</string>
     </property>
     <addaction name="actionZoom_Out" />
    </widget>
    <addaction name="actionBuildBuild_Project" />
    <addaction name="actionBuildCheck_File" />
    <addaction name="menuSelect_Target" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>Settings</string>
    </property>
    <addaction name="actionSettingsGeneral_Settings" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout" />
   </widget>
   <addaction name="menuProjekt" />
   <addaction name="menuDatei" />
   <addaction name="menuBearbeiten" />
   <addaction name="menuFenster" />
   <addaction name="menuBuild" />
   <addaction name="menuNavigatoin" />
   <addaction name="menuSettings" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFileOpen" >
   <property name="text" >
    <string>Open...</string>
   </property>
  </action>
  <action name="actionFileClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionFileSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionEditUndo" >
   <property name="text" >
    <string>Undo</string>
   </property>
  </action>
  <action name="actionEditRedo" >
   <property name="text" >
    <string>Redo</string>
   </property>
  </action>
  <action name="actionEditCut" >
   <property name="text" >
    <string>Cut</string>
   </property>
  </action>
  <action name="actionEditCopy" >
   <property name="text" >
    <string>Copy</string>
   </property>
  </action>
  <action name="actionEditPaste" >
   <property name="text" >
    <string>Paste</string>
   </property>
  </action>
  <action name="actionEditSelect_All" >
   <property name="text" >
    <string>Select All</string>
   </property>
  </action>
  <action name="actionEditComment" >
   <property name="text" >
    <string>Comment</string>
   </property>
  </action>
  <action name="actionEditUncomment" >
   <property name="text" >
    <string>Uncomment</string>
   </property>
  </action>
  <action name="actionEditToggle_Comment" >
   <property name="text" >
    <string>Toggle Comment</string>
   </property>
  </action>
  <action name="actionEditFind" >
   <property name="text" >
    <string>Find</string>
   </property>
  </action>
  <action name="actionActionEditDelete" >
   <property name="text" >
    <string>Delete</string>
   </property>
  </action>
  <action name="actionFind_in_Files" >
   <property name="text" >
    <string>Find in Files...</string>
   </property>
  </action>
  <action name="actionProjectOpen" >
   <property name="text" >
    <string>Open...</string>
   </property>
  </action>
  <action name="actionProjectNew" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionProjectSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionProjectSave_As" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionProjectClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionFileSaveAs" >
   <property name="text" >
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionFileNew" >
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionProjectAddFile" >
   <property name="text" >
    <string>Add File...</string>
   </property>
  </action>
  <action name="actionProjectAddCurrentFile" >
   <property name="text" >
    <string>Add Current File</string>
   </property>
  </action>
  <action name="actionSettingsGeneral_Settings" >
   <property name="text" >
    <string>General Settings...</string>
   </property>
  </action>
  <action name="actionHelpAbout" >
   <property name="text" >
    <string>About...</string>
   </property>
  </action>
  <action name="actionViewZoom_In" >
   <property name="text" >
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out" >
   <property name="text" >
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionViewDefault_Zoom" >
   <property name="text" >
    <string>Default Zoom</string>
   </property>
  </action>
  <action name="actionBuildBuild_Project" >
   <property name="text" >
    <string>Build Project</string>
   </property>
  </action>
  <action name="actionBuildCheck_File" >
   <property name="text" >
    <string>Check File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Mainwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>356</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
